Vagrantfile

```diff
@@ -68,27 +68,12 @@ module Forklift
         @host_vars = {box['name'] => box['ansible']['variables']}
 
         if (playbooks = box['ansible']['playbook'])
-
-          if playbooks.is_a?(String)
-            args = SUPPORT_NAMED_PROVISIONERS ? ['main', type: 'ansible'] : [:ansible]
```

Comment:
I don't see this still being supported which was added to prevent breakage of bats on our CI system where an older version of vagrant is used.

I like the code consolidation though.